src/current/_includes/releases/v23.2/v23.2.1.md

```diff
@@ -0,0 +1,161 @@
+- Fixed a bug where concurrent [`GRANT`](https://www.cockroachlabs.com/docs/v23.2/grant) statements can cause deadlocks. [#117713][#117713]
+- CockroachDB can now transparently retry more retryable errors when performing a non-atomic [`COPY`](https://www.cockroachlabs.com/docs/v23.2/copy-from) command. [cockroachdb/cockroach#117895][#117895]
+- Fixed a bug that caused [DML statements](https://www.cockroachlabs.com/docs/v23.2/performance-best-practices-overview#dml-best-practices) to fail while a [hash-sharded index](https://www.cockroachlabs.com/docs/v23.2/hash-sharded-indexes) was being created. The symptom of this bug was an error like `column "crdb_internal_val_shard_16" does not exist`. This bug was present since v23.1.0. [#118215][#118215]
+- Previously, CockroachDB could encounter the error `unable to encode table key: *tree.DTSQuery` when operating on columns with the internal `TSQuery` type in some contexts (e.g. when collecting [table statistics](https://www.cockroachlabs.com/docs/v23.2/cost-based-optimizer#table-statistics) or when performing a [`DISTINCT` operation](https://www.cockroachlabs.com/docs/v23.2/select-clause#eliminate-duplicate-rows)), and this is now fixed. The bug had been present since v23.1 when support for the internal `TSQuery` type was added. [cockroachdb/cockroach#118321][#118321]
+- Previously, in some cases CockroachDB could incorrectly evaluate queries that scanned an [inverted index](https://www.cockroachlabs.com/docs/v23.2/inverted-indexes) and had a [`WHERE` filter](https://www.cockroachlabs.com/docs/v23.2/select-clause#where-clause) in which two sides of the `AND` expression had "similar" expressions (e.g. `ARRAY['str1'] <@ col AND (ARRAY['str1'] && col OR ...)`); this is now fixed. The bug had been present since prior to v22.2. [#118360][#118360]
+- Fixed a bug that could cause [`DELETE`](https://www.cockroachlabs.com/docs/v23.2/delete) queries sent by the [row-level TTL](https://www.cockroachlabs.com/docs/v23.2/row-level-ttl) [job](https://www.cockroachlabs.com/docs/v23.2/show-schedules) to use a [secondary index](https://www.cockroachlabs.com/docs/v23.2/indexes) rather than the [primary index](https://www.cockroachlabs.com/docs/v23.2/primary-key) to find the rows to delete. This could lead to some `DELETE` operations taking a much longer time than they should. This bug was present since v22.2.0. [#118337][#118337]
+- Fixed the issue where we were missing data on SQL statistics and consequently missing data on the [SQL Activity page](https://www.cockroachlabs.com/docs/v23.2/ui-overview#sql-activity) by properly recalculating the value from the current and past hour on the top activity table. [cockroachdb/cockroach#118378][#118378]
```

Comment:
This one reads a bit unclear to me (not sure what "top activity table" means in this context).

```suggestion
- Fixed an issue with missing data on SQL statistics, and consequently missing data on the [SQL Activity page](https://www.cockroachlabs.com/docs/v23.2/ui-overview#sql-activity), by properly recalculating the value from the current and past hour on the top activity table. [cockroachdb/cockroach#118378][#118378]
```

Reply:
✅